README.md

```diff
@@ -1,14 +1,69 @@
 # Pipeline Shared Libraries
 
-When you have multiple Pipeline jobs, you often want to share some parts of the Pipeline
-scripts between them to keep Pipeline scripts [DRY](http://en.wikipedia.org/wiki/Don't_repeat_yourself).
+When you have multiple Pipeline jobs, you often want to share some parts of the Pipeline scripts between them to keep Pipeline scripts [DRY](http://en.wikipedia.org/wiki/Don't_repeat_yourself).
 A very common use case is that you have many projects that are built in the similar way.
 
-This plugin adds that functionality by allowing you to create “shared library script” SCM repositories.
-It can be used in two modes:
-a legacy mode in which there is a single Git repository hosted by Jenkins itself, to which you may push changes;
-and a more general mode in which you may define libraries hosted by any SCM in a location of your choice.
+This plugin adds that functionality by allowing you to create “shared library script” SCM repositories. It can be used in two modes:
```

Comment:
Sentence per line was used before.  Wouldn't this reduce the diffs and make it easier to see that the change here is to make two important ideas into bullet points?
```suggestion
This plugin adds that functionality by allowing you to create “shared library script” SCM repositories.
It can be used in two modes:
```